.babelrc

```diff
@@ -1,35 +1,50 @@
 {
-  presets: [
+  "presets": [
     [
-      "env",
+      "@babel/preset-env",
       {
-        targets: {
-          node: "current",
-          browsers: "last 2 chrome versions, last 2 firefox versions"
+        "targets": {
+          "node": "current",
+          "browsers": "last 2 chrome versions, last 2 firefox versions"
         },
-        modules: false
+        "modules": false
       }
     ],
-    "react",
-    "flow"
+    "@babel/preset-react",
+    "@babel/preset-flow"
   ],
-  plugins: [
-    "transform-class-properties",
-    [ "transform-object-rest-spread", { "useBuiltIns": true }]
+  "plugins": [
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
```

Comment:
"loose" was the default before. Without it we have +8kB in the result. The difference is that it uses `defineProperty` instead of direct assignments for these properties.

Reply:
Does this file support comments? If so it would be nice to have this comment in the source code.

Reply:
yep